source/agora/test/Base.d

```diff
@@ -200,9 +200,16 @@ public NetworkT makeTestNetwork (NetworkT : TestNetwork)
 
         auto net = new NetworkT(NodeConfig.init, node_configs.map!(
             c => c.key_pair.address.toString()).array);
+
+        // accessibility workaround: Base module has access to protected
+        // createNewNode because it's defined in TestNetwork in *this* module,
+        // however it cannot access NetworkT's override because it usually
+        // lives in another module
+        auto base_net = cast(TestNetwork)net;
+
         foreach (idx, ref conf; configs)
         {
-            net.apis ~= net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
+            net.apis[node_configs[idx].key_pair.address] = base_net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
```

Comment:
Bad rebase ?

Reply:
right.. well I'll need to rebase it again if https://github.com/bpfkorea/agora/pull/45 is merged first.